.travis.yml

```diff
@@ -18,5 +18,10 @@ matrix:
         directories:
           - $HOME/Library/Caches/go-build
           - $HOME/gopath/pkg/mod
+
+before_install:
+  - env GO111MODULE=off go get -u golang.org/x/lint/golint
+
 script:
+  - make ci-lint
```

Comment:
+1 to adding this to travis, see my other comment about possibly combining the 2 targets.

Reply:
Are you upvoting splitting the linting out from the build/test? Or just the general idea we should add this stuff to travis?